src/main/webapp/creatives/maps/simid_map_creative.js

```diff
@@ -50,6 +52,37 @@ export default class SimidMapCreative extends BaseSimidCreative {
    * @private 
   */
   grantLocationAccess_() {
-    //ToDo(kristenmason@): implement map
+    //ToDo(kristenmason@): implement the Google Maps request access functionality
+    this.simidProtocol.sendMessage(CreativeMessage.REQUEST_PAUSE).then(() => {
+      findNearest.classList.add("hidden");
+      //ToDo(kristenmason@): display map
+      this.createButtons_();
+    })
+  }
+
+  createButtons_() {
+    const returnToAdButton = document.createElement("button");
+    returnToAdButton.innerHTML = "Return To Ad";
+    returnToAdButton.id = "returnToAd";
+    returnToAdButton.onclick = () => this.playAd_(returnToAdButton); 
+
+    const skipToContentButton = document.createElement("button");
+    skipToContentButton.innerHTML = "Skip To Content";
```

Comment:
Most skip buttons just say "Skip" and a user understands what that is, perhaps you could say "Skip Ad" if you wanted.

Reply:
okay -- I can change to Skip Ad

Reply:
Just to add to that -- you want to avoid "Skip To Content", because you don't know for sure that this is the last ad.  Its possible that this ad is part of a pod containing a few ads, a user would be confused if the click "Skip to Content" just to then be shown the second ad in the pod